gulpfile.ts

```diff
@@ -97,45 +128,181 @@ function buildTLEGrammar(): void {
 }
 
 async function buildGrammars(): Promise<void> {
-    if (!fs.existsSync('dist')) {
-        fs.mkdirSync('dist');
+    if (!fse.existsSync('dist')) {
+        fse.mkdirSync('dist');
     }
-    if (!fs.existsSync('dist/grammars')) {
-        fs.mkdirSync('dist/grammars');
+    if (!fse.existsSync('dist/grammars')) {
+        fse.mkdirSync('dist/grammars');
     }
 
     buildTLEGrammar();
 
-    fs.copyFileSync(jsonArmGrammarSourcePath, jsonArmGrammarDestPath);
+    fse.copyFileSync(jsonArmGrammarSourcePath, jsonArmGrammarDestPath);
     console.log(`Copied ${jsonArmGrammarDestPath}`);
-    fs.copyFileSync(armConfigurationSourcePath, armConfigurationDestPath);
+    fse.copyFileSync(armConfigurationSourcePath, armConfigurationDestPath);
     console.log(`Copied ${armConfigurationDestPath}`);
 }
 
-// tslint:disable-next-line:no-suspicious-comment
-// TODO: This is temporary - retrieves binaries from an internal location to package with the extension
-async function updateLanguageServer(): Promise<void> {
-    // tslint:disable-next-line: max-line-length
-    // let armServerBin = path.join(env.ExtensionsBin, '..', '..', 'ARM-LanguageServer', 'Microsoft.ArmLanguageServer', 'bin', 'Debug', 'netcoreapp2.2', 'publish');
-    let armServerBin = '\\\\scratch2\\scratch\\stephwe\\ARM\\Dev Assemblies\\Current';
-    let updateDest = path.join(__dirname, languageServerFolderName);
-    if (!fs.existsSync(updateDest)) {
-        fs.mkdirSync(updateDest);
+function executeInShell(command: string): void {
+    console.log(command);
+    const result = shelljs.exec(command);
+    console.log(result.stdout);
+    console.log(result.stderr);
```

Comment:
Is this writing to the terminal? Is there a reason we don't use ext's `outputChannel` here?

Reply:
This is gulp, thus during build.  No output channel.